release/trigger-publish

```diff
@@ -1,2 +1 @@
-1.22.5
-
```

Comment:
donot remove the newline

Reply:
it's fine, that was there because they published twice last time.